opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -365,12 +365,12 @@ enum AggregationTemporality {
 //   (point.flags & FLAG_NO_RECORDED_VALUE) == FLAG_NO_RECORDED_VALUE
 //
 enum DataPointFlags {
-  FLAG_NONE = 0;
+  DATA_POINTS_FLAG_NONE = 0;
 
   // This DataPoint is valid but has no recorded value.  This value
   // SHOULD be used to reflect explicitly missing data in a series, as
   // for an equivalent to the Prometheus "staleness marker".
-  FLAG_NO_RECORDED_VALUE = 1;
+  DATA_POINTS_FLAG_NO_RECORDED_VALUE = 1;
```

Comment:
```suggestion
  DATA_POINTS_FLAGS_NO_RECORDED_VALUE = 1;
```

If we're going to change these we should be consistent with the `LOG_RECORD_FLAGS_*` values.

Reply:
@tigrannajaryan my point on removing seems to be proven here. I `DATA_POINTS_FLAGS_NO_RECORDED_VALUE` sounds like the value of the "flags" not the value of the "mask" that you need to "bit and/or". I do suggest even more to remove them.